### ISSUE TITLE
Missing accessible name in TextfiledWrapper type="search" component

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2168]
Description  
---
Our TextfieldWrapper component with Input `type="search"` has due to Browser consistency a custom "clear" control button. This button misses an accessible name which leads to an error in AXE Devtools automated accessibility testing.

Bug reported by: https://porschedev.slack.com/archives/CT7AVHTTQ/p1668000322669309 

To Reproduce
---
1. Go to https://designsystem.porsche.com/v2/components/text-field/examples#type-search
2. Type something into the field
3. Check the (missing) accessible name of the clear ("x") button in your DevTools

Expected behavior
---
- [x] Due it's not really relevant for screen reader users it should have an accessible name (like "clear field")
- [x] `aria-hidden` used for clear button is removed

Test Environment
---
Platform/Browser/Version: all
